README.md

```diff
@@ -1 +1,2 @@
 # movie-archive
+# By
```

Comment:
Same here. Something is wrong. These changes are already in your main branch

Reply:
I know. I mean you did this before and have already merged this change into the main branch. Why is it showing again in this PR?